src/PIL/ImageGrab.py

```diff
@@ -29,16 +29,18 @@
     raise ImportError("ImageGrab is macOS and Windows only")
 
 
-def grab(bbox=None, include_layered_windows=False):
+def grab(bbox=None, include_layered_windows=False, multimonitor=False):
```

Comment:
I just had a thought, would it be better to rename the `multimonitor` parameter to `primary_only` (and invert its value)?

Reply:
I think the problem with inverting it would be that this then changes the default behaviour - Pillow values backwards compatibility.

If you feel like the name needs to clearer, I might suggest `all_screens`.

Reply:
I was thinking about making the default be None, which would use the old (platform dependent) behaviour, but could be overriden with True or False.

Reply:
Alright, I got my hands on a mac with multiple monitors; it turns out I was wrong - mac also captures the primary monitor only.

I will rename the parameter to `all_screens` as you suggested.